hazelcast/src/main/java/com/hazelcast/client/impl/statistics/ClientStatisticsService.java

```diff
@@ -106,6 +106,10 @@ public final void start() {
         logger.info("Client statistics is enabled with period " + periodSeconds + " seconds.");
     }
 
+    public void onConnectionToNewCluster() {
```

Comment:
Maybe we can rename this to `collectAndSendStatsNow`

Reply:
Just seen this. Sure why not.